docs/application/native/guides/connectivity/nfc.md

```diff
@@ -98,9 +98,15 @@ The main features of the NFC API include:
 
   ![HCE routing](./media/nfc_hce_routing.png)
 
+  > **Note**
+  >
+  > - HCE operations begin with the launch of HCE application in Tizen Application Framework. With the execution of these operations, the data transfer to the application needs more time.
+  > - In case, you consider time efficiency, it is recommend not to create HCE applications and communicate with the Secure Elements provided in the device.
```

Comment:
@nawab-samsung How about keeping "If" instead of "In case,"?

If you consider time efficiency, ~

Reply:
@mijin85cho 
I wrote "incase" because there might be cases where we think about time efficiency ,else we do not consider the time efficiency.

The thing is when we think of time efficiency , that is a certain condition/criteria associated as: "when" "why"

Reply:
@nawab-samsung I'm suggesting "if" because of the same reason with your comment.
"if you consider the time efficiency" goes with "else, you don't consider". 

With "In case, you consider time efficiency, ", I thought there are too many commas than required, stopping the flow of the sentence without meaning. Of course the condition is important, but more simple word would be better.